### DISCUSSION TITLE
Why are some shortcuts rejected ?

### DISCUSSION THREAD

**Discussion by HadrienMP** [F1bonacc1/process-compose#140] [Q&A]
I am trying to customize the shorcuts because I can't really use the function keys. I was trying to use : u for start and d for down, on CTRL-U and CTRL-D, but they don't seem to work. I was able to use F1 for stop though (which collides with the default "Shortcuts" shortcut)

**Comment by F1bonacc1** [comment]
Hi,

I can probably do a better job documenting this, I currently rely on a third party for valid key definitions.
Please check it out here:
https://github.com/gdamore/tcell/blob/0393f5eb0b1a9e741d389557de013894d85d4944/key.go#L83
